feature/automated-releases.feature

```diff
@@ -23,39 +27,56 @@ Feature: Automated releases
 
   Scenario: If a major release branch exists, the tool creates a major release from there
     Given following existing branches:
-      | name   |
-      | 1.0.x  |
-      | master |
+      | name    |
+      | 1.0.x   |
+      | 2.0.x   |
+      | master  |
+      | develop |
     And following open milestones:
       | name  |
       | 2.0.0 |
     When I close milestone "2.0.0"
     Then tag "2.0.0" should have been created on branch "master"
     And branch "2.0.x" should have been created from "master"
 
-  Scenario: If a major release branch exists, the tool creates a new minor release from there
+  Scenario: If a new major release branch exists, the tool does not create a new minor release
     Given following existing branches:
-      | name   |
-      | 1.0.x  |
-      | master |
+      | name    |
+      | 1.0.x   |
+      | 2.0.x   |
+      | master  |
+      | develop |
     And following open milestones:
       | name  |
       | 1.1.0 |
     When I close milestone "1.1.0"
-    Then tag "1.1.0" should have been created on branch "master"
-    And branch "1.1.x" should have been created from "master"
+    Then the tool should have halted with an error
```

Comment:
I'm assuming this is because the `1.1.x` branch does not exist? (just want a clarification)

Reply:
Correct: no auto-magic branch switching 👍